library/leakcanary-android/src/main/java/com/squareup/leakcanary/AndroidExcludedRefs.java

```diff
@@ -252,6 +252,15 @@
       excluded.staticField("android.media.AudioManager", "mContext_static");
     }
 
+    if (SDK_INT > ICE_CREAM_SANDWICH && SDK_INT <= LOLLIPOP_MR1) {
```

Comment:
Can you move this up? I like to leave the pure AOSP ones above the manufacturer ones. That's just for the show when someone first reads the code :) .